tools/pages/admin/requests/queries.ts

```diff
@@ -117,3 +117,29 @@ export const GET_APPLICANT_RENEWAL_QUERY = gql`
     }
   }
 `;
+
+export const GET_APPLICANT_REPLACEMENT_QUERY = gql`
```

Comment:
This variable name is not sufficient to describe what this query does. Could you please add a JSDoc comment above with clarification?